cypress-tests/cypress/fixtures/confirm-body.json

```diff
@@ -2,8 +2,6 @@
   "client_secret": "",
   "return_url": "https://hyperswitch.io",
   "confirm": true,
-  "payment_method": "card",
```

Comment:
What is the reason for removing payment method from fixtures and adding individually to each configs? Otherwise LGTM

Reply:
For corner cases, especially for `confirm payment w/o pmd`, we're not supposed to pass any `payment_method` info. With `payment_method` being existent in `confirm_body` fixture, it is being supplied in the request resulting in failures.  
To over come this, I had to remove it from fixtures which resulted in other tests to fail forcing me to add at config level.